apps/ledger-live-mobile/src/families/tezos/operationDetails.ts

```diff
@@ -1,7 +1,8 @@
 import type { Operation } from "@ledgerhq/types-live";
 
 export default {
-  getURLWhatIsThis: (op: Operation) =>
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why adding an unsused argument ?

Reply:
Similar to the argument in getURLFeesInfo method. I need it in cosmos family